inference-engine/tests/functional/plugin/gpu/subgraph_tests/shared_constant.cpp

```diff
@@ -33,6 +33,9 @@ class SharedConstant : virtual public LayerTestsUtils::LayerTestsCommon {
         auto mul = std::make_shared<opset8::Multiply>(input4, constant);
         auto input5 = std::make_shared<opset8::Parameter>(type, constGroupConvBackpropShape);
         auto mul2 = std::make_shared<opset8::Multiply>(input5, constantGroupConv);
+        // explicitly set the output name, to avoid global conflict
+        mul2->set_friendly_name("Multiply_0");
```

Comment:
is this change really relevant to the PR?

Reply:
actually this is a bug in this case, it will compare the outputs in the order of output map, not by retrieving the value from map key. with my change adding some test in GPU, it will trigger this bug, because the global node name for each ngraph node will change with more case added, thus will fail when comparing the outputs. I'm fixing it by setting the output names explicitly instead of letting the ie/ngraph generate it.